### ISSUE TITLE
Bad results for pose2body at 256 resolution

### ISSUE THREAD

**Issue by tharindu-mathew** [NVIDIA/vid2vid#96]
I'm trying to train the pose model, and verify the output for the 256 resolution. But, after I run train_256.sh with the following options the output looks quite bad (I had to change options so it doesn't run out of memory on a Titan xp). I did notice the training images have only 100 images, which clearly is a few seconds of footage not 3 to 4 minutes.

Could you suggest on what to do to get sharp images at a 256 resolution? Thanks in advance.

Image:
[https://imgur.com/a/QkPTWus](url)

Script:
$ cat ./scripts/pose/train_256p.sh
#!/bin/bash
python train.py --name pose2body_256p --dataroot datasets/pose \
--dataset_mode pose --input_nc 6 --num_D 1 \
--resize_or_crop randomScaleHeight_and_scaledCrop --loadSize 384 \
--fineSize 256 --gpu_ids 2,3 --batchSize 1 \
--max_frames_per_gpu 3 --niter 5 \
--niter_decay 5 --no_first_img \
--n_frames_total 2 --max_t_step 4

**Comment by justfouw** [comment]
@tharindu-mathew 
The training videos for pose experiment are not provided. The one video in dataset/pose is just for example usage. It is necessary to know the number of training/val/ pose dataset, if we want to reimplement the result reported in the paper. I have summit one issue about this point #98

**Comment by tharindu-mathew** [comment]
So, if I try with a more complete dataset, it should provide reasonable results? Is that what you're implying. I think the videos used by them are on youtube.

**Comment by justfouw** [comment]
@tharindu-mathew Yes, more dataset, more reasonable results. Do you have idea about this kinds of dataset, such as solo dance..
